package.json

```diff
@@ -168,6 +168,6 @@
   },
   "engines": {
     "node": "2.5",
-    "npm": "3.2"
+    "npm": ">=2"
```

Comment:
This still matches npm 3, no? We should probably lock in the version we expect, or at least use 2.x